lib/game.rb

```diff
@@ -0,0 +1,72 @@
+require_relative "../config/environment.rb"
+
+class Game
+
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+  ]
+
+  def current_player
+    board.cells.select{|cell| !cell.strip.empty?}.count.even? ? player_1 : player_2
+  end
+
+  def winner
+    winning_combo = WIN_COMBINATIONS.detect do |combo|
+      [board.cells[combo[0]], board.cells[combo[1]], board.cells[combo[2]]].uniq.count == 1 && !board.cells[combo[0]].strip.empty?
+    end
+    winning_combo.nil? ? nil : board.cells[winning_combo[0]]
+  end
+
+  def won?
+    !winner.nil?
```

Comment:
Your `won?` method should be the one that returns an array of the winning combo if there is one...nil otherwise. 

```ruby
  def won?
    WIN_COMBINATIONS.detect do |combo|
      @board.cells[combo[0]] == @board.cells[combo[1]] &&
      @board.cells[combo[0]] == @board.cells[combo[2]] &&
      @board.taken?(combo[0] + 1)
    end
  end
```